codelabs/keyboard_navigation/keyboard_navigation.md

```diff
@@ -314,12 +282,46 @@ Open the sample app and drag a function block on to your workspace. Press **ctrl
 
 ![The cursor flashing red](./flashing_cursor.gif)
 
+## Define and set a custom cursor
+
+Create a new directory at `src/cursors` and add a file inside named `custom.js`.
+
+At the top of the new file, you first need to import `blockly/core`:
+
+```js
+import * as Blockly from 'blockly/core';
+```
+
+Then define your new custom cursor and have it extend the base cursor, `Blockly.Cursor`:
+
+```js
+export class CustomCursor extends Blockly.Cursor {
+  constructor() {
+    super();
+  }
+}
+```
+
+Import your cursor at the top of `src/index.js`.
+
+```js
+import {CustomCursor} from './cursors/custom';
+```
+
+Somewhere after the existing code that injects the workspace, use its `MarkerManager` to set your new custom cursor:
+
+```js
+// Add CustomCursor to workspace
+ws.getMarkerManager().setCursor(new CustomCursor());
+```
+
 ## Change the cursor behavior
 
 ### Override the move methods
+
 In order to create a cursor that skips over previous and next connections you have to override the methods that move the cursor.
 
-Add the following code to `cursors/custom.js`, inside the `CustomCursor` class definition:
+Add the following code to `cursors/custom.js`, inside the `CustomCursor` class definition. These implementations are just a starting point that you will be improving in the next step:
```

Comment:
Colon should immediate precede the thing referred to, so either move the added sentence below the code, or make it a parenthetical:
```suggestion
Add the following code to `cursors/custom.js`, inside the `CustomCursor` class definition (these implementations are just a starting point that you will be improving in the next step):
```
also, perhaps "a starting point that will be improved in later steps"?